### PR TITLE
api gateway school connection

### PR DESCRIPTION
**Pull request by nmejiab**
## Background

Create the queries and mutations referent to the ms-school dependencies bye editing the schema, mutations and queries files, also add the documentation in the files "graphiQL-test.txt" and "query-variables.txt"

## Changes done

queries and mutations to the tables site, faculty and career.

## Pending to be done

Nothing

## Notes

No

## Demo

Result of making a get to all the tables after create some fields in each one by the GraphQl methods:
image

### CODE REVIEW

File: ag-campus-kid/src/microservices/school/mutations.js [hunk 1]
```diff
@@ -5,7 +5,36 @@ const { HttpProvider } = require("../../providers");
...
+  updateFaculty: async (root, { data }) => {
+    return await HttpProvider.put(`${baseUrl}/api/faculties`, data);
+  },
+  updateCareer: async (root, { data }) => {
+    return await HttpProvider.put(`${baseUrl}/api/careers`, data);
+  },
+
+  //Deleters
+  deleteSites: async (root, { id }) => {
+    return await HttpProvider.deleted(`${baseUrl}/api/sites/` + id);
```
Comment (jhonatan247) [inline ag-campus-kid/src/microservices/school/mutations.js]:
```suggestion
    return await HttpProvider.deleted(`${baseUrl}/api/sites/${id}`);
```

File: ag-campus-kid/src/microservices/school/mutations.js [hunk 2]
```diff
@@ -5,7 +5,36 @@ const { HttpProvider } = require("../../providers");
...
+  },
+  deleteFaculties: async (root, { id }) => {
+    return await HttpProvider.deleted(`${baseUrl}/api/faculties/` + id);
```
Comment (jhonatan247) [inline ag-campus-kid/src/microservices/school/mutations.js]:
```suggestion
    return await HttpProvider.deleted(`${baseUrl}/api/faculties/${id}`);
```

File: ag-campus-kid/src/microservices/school/mutations.js [hunk 3]
```diff
@@ -5,7 +5,36 @@ const { HttpProvider } = require("../../providers");
...
+  },
+  deleteCareers: async (root, { id }) => {
+    return await HttpProvider.deleted(`${baseUrl}/api/careers/` + id);
```
Comment (jhonatan247) [inline ag-campus-kid/src/microservices/school/mutations.js]:
```suggestion
    return await HttpProvider.deleted(`${baseUrl}/api/careers/${id}`);
```

**Review by jhonatan247 — CHANGES_REQUESTED** [review]
Good job, just some small comments.

**Comment by nmejiab** [comment]
Fix done

**Review by jhonatan247 — APPROVED** [review]

**Pull request merged**